jquery.signaturepad.js

```diff
@@ -429,11 +426,17 @@ function SignaturePad (selector, options) {
       })
 
       canvas.each(function () {
-        this.ontouchstart = null
+        this.removeEventListener('touchstart', startTouchDrawing)
       })
     }
   }
+  function startTouchDrawing {
```

Comment:
There is a syntax error. This should be `function startTouchDrawing(e) {`

Reply:
thanks for noticing that, I think I was asleep yesterday when I did that.
I added that